bitnami/thanos/Chart.yaml

```diff
@@ -15,6 +15,6 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 1.3.4
+version: 1.3.5
```

Comment:
As it is adding a feature, I would bump the minor version instead

Reply:
Will change it to 1.4.0